examples/ov5640_stopmotion_kaluga1_3.py

```diff
@@ -235,7 +235,26 @@ def take_stop_motion_gif(n_frames=10, replay_frame_time=0.3):
             g.add_frame(frame, replay_frame_time)
             for i in range(1, n_frames):
                 print(f"{i}/{n_frames}")
-                old_frame.blit(0, 0, frame, x1=0, y1=0, x2=frame.width, y2=frame.height)
+
+                # CircuitPython Versions <= 8.2.0
+                if hasattr(old_frame, "blit"):
+                    old_frame.blit(
+                        0, 0, frame, x1=0, y1=0, x2=frame.width, y2=frame.height
+                    )
+
+                # CircuitPython Versions >= 9.0.0
+                elif hasattr(bitmaptools, "blit"):
```

Comment:
I did consider it, but got a bit too caught up in the thought about there being possible cases that wouldn't match either. I lost sight of the actual behavior related to it and didn't think about the error raised actually being more helpful in that situation.

The latest commit changes it to else so that it will raise that AttributeError.

Theoretically it could use a python implementation for the `blit()` if it were running on a system without either version. But the timing hit it would take looks potentially problematic for this use-case.